### PR TITLE
chore: skip system-integration scenarios for Beats

### PR DESCRIPTION
**Pull request by mdelapenya**
- chore: copy default profile for beats
- chore: remove system integrations tests for Beats

## What does this PR do?
It copies the default profile for running the e2e tests, removing the system-integration tests, as they are adding certain flakiness

## Why is it important?
Having a more stable test execution is critical for team success, and the system integration test suite is behaving with certain flakiness, which the product team is currently investigating.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the Unit tests (`make unit-test`), and they are passing locally
- [ ] I have run the End-2-End tests for the suite I'm working on, and they are passing locally
- [ ] I have noticed new Go dependencies (run `make notice` in the proper directory)

## Related issues

- Completes https://github.com/elastic/beats/pull/31531

### CODE REVIEW

File: .ci/.e2e-tests-beats.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      - name: "Integrations"
+        tags: "integrations"
+        platforms: ["debian_arm64", "debian_amd64", "sles15"]
+      - name: "APM Integration"
+        tags: "apm_server"
+        platforms: ["debian_amd64"]
+      - name: "Linux Integration"
+        tags: "linux_integration"
+        platforms: ["debian_arm64", "debian_amd64", "sles15"]
+      # - name: "Permissions Output change"
```
Comment (cmacknz) [inline .ci/.e2e-tests-beats.yaml:29]: These look to be commented out everywhere, can they just be deleted?

File: .ci/.e2e-tests-beats.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      - name: "APM Server"
+        tags: "apm-server"
+        platforms: ["debian_amd64"]
+      - name: "Filebeat"
+        tags: "filebeat"
+        platforms: ["debian_amd64"]
+      - name: "Metricbeat"
+        tags: "metricbeat"
+        platforms: ["debian_amd64"]
+  - suite: "fleet"
```
Comment (cmacknz) [inline .ci/.e2e-tests-beats.yaml:15]: Is the base for this from https://github.com/elastic/e2e-testing/blob/main/.ci/.e2e-tests.yaml?

Can we maybe reference that file in a comment here, and note what the differences are and why?

Comment (mdelapenya) [reply .ci/.e2e-tests-beats.yaml:15]: About the differences, I'd like the product team to be the ones to define what to run when beats triggers the e2e for Fleet. I could see it defining different profiles: for kubernetes autodiscover, metricbeat, filebeat, etc.

**Review by kuisathaverat — APPROVED** [review]

File: .ci/.e2e-tests-beats.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        platforms: ["debian_amd64"]
+      - name: "Linux Integration"
+        tags: "linux_integration"
+        platforms: ["debian_arm64", "debian_amd64", "sles15"]
+      # - name: "Permissions Output change"
+      #   tags: "permission_change"
+      #   platforms: ["debian_arm64", "debian_amd64", "sles15"]
+      # - name: "Stand-alone"
+      #   tags: "stand_alone_agent"
+      #   platforms: ["fleet_elastic_pkg"]
```
Comment (mdelapenya) [inline .ci/.e2e-tests-beats.yaml:34]:
```suggestion
```

**Pull request merged**